frontend/simple/views/Contributions.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <main>
+    <div class="section is-hero">
+      <i18n tag="h1" class="title is-1 is-marginless">
+        Contributions
+      </i18n>
+      <i18n tag="p">See everyone’s contributions to the group and manage yours.</i18n>
+    </div>
+
+    <div class="section columns is-desktop is-multiline c-grid">
+      <section class="column">
+        <i18n tag="h2" class="title is-3">Receiving</i18n>
+
+        <ul class="c-ul">
+          <contribution
+            v-for="contribution, index in receiving"
+            :variant="isMonetary(contribution.type) ? 'editable' : undefined"
+            v-on="isMonetary(contribution.type) ? { 'control-click': showReceivingSettings } : {}"
+          >
+            <i18n tag="strong"v-if="isMonetary(contribution.type)"
+              :args="{ currency, amount: receiving[index].what}"
+            >{currency}{amount} for mincome</i18n>
+            <strong v-else>{{contribution.what}}</strong>
+            <i18n>from</i18n>
+            {{getWho(contribution.who)}}
```

Comment:
So is there a reason that lines 23, 24, and 25 can't be placed into the i18n tag on line 20? Remember that at least entire sentences must be translated as a whole, not as parts, as otherwise it might be impossible to translate the UI into some language

Reply:
I think you didn't read another comment I've done yesterday:

> Here we're dealing with text within html tags and part of it has a button that is a tooltip... Kinda tricky. @taoeffect how do you suggest to implement the translation here?

Reply:
Maybe you didn't see my last comment yesterday asking for you advice on the implementation for this exact lines:

> Here we're dealing with text within html tags and part of it has a button that is a tooltip... Kinda tricky. @taoeffect how do you suggest to implement the translation here?

So, instead of having _**Cooking** from John_ we'll have _<\strong>Cooking<\/strong> from John_.

Reply:
Actually I've found a way using Vue native directive `v-html` [Raw-HTML](https://vuejs.org/v2/guide/syntax.html#Raw-HTML)... but we still have the button/tooltip (_and X others_ part) out of i18n... check the last commit and let me know your thoughts.